crates/bundler/src/mock_relay.rs

```diff
@@ -0,0 +1,122 @@
+use anyhow;
+use async_trait::async_trait;
+use ethers::{
+    middleware::SignerMiddleware,
+    prelude::LocalWallet,
+    providers::{Http, JsonRpcClient, Middleware, Provider},
+    types::{Bytes, TxHash, H256, U256, U64},
+};
+use ethers_flashbots_test::{
+    relay::SendBundleResponse, BundleRequest, BundleTransaction, PendingBundle, SimulatedBundle,
+    SimulatedTransaction,
+};
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use std::sync::Arc;
+use std::time::Duration;
+
+pub const INIT_BLOCK: u64 = 17832062;
+
+#[rpc(server, namespace = "eth")]
+pub trait MockFlashbotsRelay {
```

Comment:
Is this needed only in tests?

Then we can add #[test]

Reply:
The mock_relay module has been marked as test
https://github.com/qi-protocol/silius/blob/24afb5ca8d1b095bf62fedc4b2b7eee14b2ce2f9/crates/bundler/src/lib.rs#L5

Will this suffice?